documentation/onboard-dpg-in-sdkautomation/Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md

```diff
@@ -0,0 +1,31 @@
+# Integrate DPG and ApiView in SDK Automation Pipeline
+This document is targeting for everyone who wants to know all the implementation details of SDK automation pipeline.
+It describes the details of integrating DPG and ApiView in SDK Automation Pipeline.
+
+Before go through this document, please go through [Service Onboard DPG with Swagger CI Pipeline](README.md) to be familiar with the user experience in service team side. Also, you need to be familiar with [SDK Automation Pipeline Framework](../sdkautomation/README.md).
+
+# WorkFlow
+![integrate-dpg-and-apiview](integrate-dpg-and-apiview.png)
+
+__Description:__
+1. There is swagger PR triggering the SDK Automation Pipeline.
+2. SDK Automation Pipeline Framework checks whether there is a branch named `swagger/<prNumber>` in sdk repository. If yes, the branch `swagger/<prNumber>` will be used as base branch to generate SDK. Otherwise, `main` branch will be used as base branch.
+   1. `<prNumber>` is the swagger PR number.
+   2. If there is branch `swagger/<prNumber>`, `main` branch is usually used and sometimes not. It depends on the `mainRepository` defined in [specificationRepositoryConfiguration](../../specificationRepositoryConfiguration.json).
+3. SDK automation script will search for the corresponding autorest configuration file in sdk repository. If those files are not found, the pipeline automation script will output `{"packages": []}` in `generateOutput.json`, and stop the generation.
```

Comment:
In the case were we don't find an autorest.md we should consider adding a message with link to instructions on how to get a SDK generated.

Reply:
please review the latest doc.